test/kldivergences.jl

```diff
@@ -0,0 +1,69 @@
+@testset "KL divergences" begin
+    function test_kl(p, q)
+        @test kldivergence(p, q) > 0
```

Comment:
Maybe also check that `kldivergence(p, p)` and `kldivergence(q, q)` is approximately zero?